migration/391lts-392lts/04450_2347_FixWVPaymentReduceDupCode.xml

```diff
@@ -0,0 +1,426 @@
+      <PO AD_Table_ID="119" Action="I" Record_ID="0" Table="AD_Message_Trl">
+        <Data AD_Column_ID="609" Column="Created">2019-03-02 07:09:28.167</Data>
+        <Data AD_Column_ID="611" Column="Updated">2019-03-02 07:09:28.167</Data>
+        <Data AD_Column_ID="608" Column="IsActive">true</Data>
+        <Data AD_Column_ID="343" Column="MsgTip" isNewNull="true"/>
+        <Data AD_Column_ID="1192" Column="AD_Client_ID">0</Data>
+        <Data AD_Column_ID="1193" Column="AD_Org_ID">0</Data>
+        <Data AD_Column_ID="610" Column="CreatedBy">100</Data>
+        <Data AD_Column_ID="344" Column="IsTranslated">true</Data>
+        <Data AD_Column_ID="341" Column="AD_Language">es_MX</Data>
+        <Data AD_Column_ID="6767" Column="AD_Message_ID">53570</Data>
+        <Data AD_Column_ID="612" Column="UpdatedBy">100</Data>
+        <Data AD_Column_ID="84347" Column="UUID" isNewNull="true"/>
+        <Data AD_Column_ID="342" Column="MsgText">Diario de Caja</Data>
+      </PO>
+    </Step>
+    <Step SeqNo="40" StepType="AD">
+      <PO AD_Table_ID="109" Action="I" Record_ID="53571" Table="AD_Message">
+        <Data AD_Column_ID="588" Column="IsActive">true</Data>
+        <Data AD_Column_ID="589" Column="Created">2019-03-02 07:22:15.89</Data>
+        <Data AD_Column_ID="198" Column="MsgText">Cash Journal</Data>
+        <Data AD_Column_ID="591" Column="Updated">2019-03-02 07:22:15.89</Data>
+        <Data AD_Column_ID="197" Column="MsgType">I</Data>
+        <Data AD_Column_ID="6766" Column="Value">PaymentFormController: CashJournalTip</Data>
+        <Data AD_Column_ID="199" Column="MsgTip">Select the Cash Journal Definition to use.  Any cash payment created will be added to a Cash Closing document of the selected Cash Journal.</Data>
+        <Data AD_Column_ID="392" Column="AD_Org_ID">0</Data>
+        <Data AD_Column_ID="391" Column="AD_Client_ID">0</Data>
+        <Data AD_Column_ID="7716" Column="EntityType">D</Data>
+        <Data AD_Column_ID="6765" Column="AD_Message_ID">53571</Data>
+        <Data AD_Column_ID="590" Column="CreatedBy">100</Data>
+        <Data AD_Column_ID="592" Column="UpdatedBy">100</Data>
+        <Data AD_Column_ID="84346" Column="UUID" isNewNull="true"/>
+      </PO>
+    </Step>
+    <Step SeqNo="50" StepType="AD">
+      <PO AD_Table_ID="119" Action="I" Record_ID="0" Table="AD_Message_Trl">
+        <Data AD_Column_ID="342" Column="MsgText">Diario de Caja</Data>
+        <Data AD_Column_ID="344" Column="IsTranslated">true</Data>
+        <Data AD_Column_ID="609" Column="Created">2019-03-02 07:22:17.195</Data>
+        <Data AD_Column_ID="611" Column="Updated">2019-03-02 07:22:17.195</Data>
+        <Data AD_Column_ID="608" Column="IsActive">true</Data>
+        <Data AD_Column_ID="343" Column="MsgTip">Seleccione la definici&oacute;n de Diario de Caja a utilizar. Cualquier pago en efectivo creado se agregar&aacute; a un documento de Cierre de Caja del Diario de Caja seleccionado.</Data>
```

Comment:
Please fix it
<pre>
[Fatal Error] 04450_2347_FixWVPaymentReduceDupCode.xml:65:80: The entity "oacute" was referenced, but not declared.
org.xml.sax.SAXParseException; systemId: file:/opt/Development/workspace/adempierePullRequest/migration/391lts-392lts/04450_2347_FixWVPaymentReduceDupCode.xml; lineNumber: 65; columnNumber: 80; The entity "oacute" was referenced, but not declared.
	at org.apache.xerces.parsers.DOMParser.parse(Unknown Source)
	at org.apache.xerces.jaxp.DocumentBuilderImpl.parse(Unknown Source)
	at javax.xml.parsers.DocumentBuilder.parse(DocumentBuilder.java:205)
	at org.compiere.process.MigrationFromXML.loadFile(MigrationFromXML.java:133)
	at org.compiere.process.MigrationFromXML.loadXML(MigrationFromXML.java:107)
	at org.compiere.process.MigrationFromXML.doIt(MigrationFromXML.java:49)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:175)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:128)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:160)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:105)
	at org.compiere.apps.ProcessCtl.startProcess(ProcessCtl.java:623)
	at org.compiere.apps.ProcessCtl.run(ProcessCtl.java:372)
	at java.lang.Thread.run(Thread.java:745)
</pre>